src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static Boolean isFinished = false;
+    private static int itemCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        Task[] items = new Task[100];
+
+        printIntro();
+
+        while (!isFinished) {
```

Comment:
Perhaps for this part down, you could refractor it as a method, so your main class will not be too long?